tests/composition/test_composition.py

```diff
@@ -2641,6 +2641,38 @@ def test_LPP_wrong_component(self):
                 and "to InputPort" in str(error_text.value)
                 and "that is in deferred init" in str(error_text.value))
 
+    @pytest.mark.composition
+    @pytest.mark.parametrize("mode", ['Python',
+                                      pytest.param('LLVM', marks=pytest.mark.llvm),
+                                      pytest.param('LLVMExec', marks=pytest.mark.llvm),
+                                      pytest.param('LLVMRun', marks=pytest.mark.llvm),
+                                      pytest.param('PTXExec', marks=[pytest.mark.llvm, pytest.mark.cuda]),
+                                      pytest.param('PTXRun', marks=[pytest.mark.llvm, pytest.mark.cuda])
```

Comment:
Please drop `*Run` from the list. It doesn't do anything for `execute`

Reply:
Updated